pkg/asset/manifests/kube-core-operator.go

```diff
@@ -113,7 +114,11 @@ func (kco *kubeCoreOperator) getAPIServerURL() string {
 }
 
 func (kco *kubeCoreOperator) getEtcdServersURLs() string {
-	return fmt.Sprintf("https://%s-etcd.%s:2379", kco.installConfig.Name, kco.installConfig.BaseDomain)
+	var urls []string
+	for i := 0; i < kco.installConfig.MasterCount(); i++ {
+		urls = append(urls, fmt.Sprintf("https://%s-etcd-%d.%s:2379", kco.installConfig.Name, i, kco.installConfig.BaseDomain))
+	}
+	return strings.Join(urls, ",")
```

Comment:
I would much rather return `[]string` here and only `Join` where that serialization is needed.